src/components/PeopleToDisplay.tsx

```diff
@@ -0,0 +1,84 @@
+import { useMemo } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { Person } from '../types';
+import { SortFields } from '../types/SortFields';
+
+const useQueryFilter = (query: string | null, people: Person[]): Person[] => {
+  if (!query) {
+    return people;
+  }
+
+  const searchedQuery = query.toLowerCase();
+
+  return people.filter(
+    (person) => person.name.toLowerCase().includes(searchedQuery)
+      || person.sex.toLowerCase().includes(searchedQuery),
+  );
+};
+
+const useSexFilter = (sex: string, people: Person[]): Person[] => {
+  if (!sex) {
+    return people;
+  }
+
+  return people.filter((person) => person.sex === sex);
+};
+
+const useCenturiesFilter = (
+  centuries: string[],
+  people: Person[],
+): Person[] => {
+  if (centuries.length === 0) {
+    return people;
+  }
+
+  return people.filter((person) => centuries
+    .includes(Math.ceil(Number(person.born) / 100).toString()));
+};
+
+const useSort = (
```

Comment:
`use` prefix is used for hooks only, in your case, they are just utils/helpers functions.
Rename and move them to utils/helpers file or folder